Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -528,8 +556,23 @@ extension CryptoStore: BraveWalletKeyringServiceObserver {
     rejectAllPendingWebpageRequests()
   }
   public func keyringCreated(_ keyringId: String) {
+    Task { @MainActor in
+      var coin: BraveWallet.CoinType?
+      for supportedCoin in WalletConstants.supportedCoinTypes where supportedCoin.keyringId == keyringId {
+        coin = supportedCoin
+        break
+      }
+      if let newCoin = coin {
+        migrateUserAssets(for: newCoin)
+      }
```

Comment:
Essentially the same but easier to read:
```suggestion
      if let newCoin = WalletConstants.supportedCoinTypes.first(where: { $0.keyringId == keyringId }) {
        migrateUserAssets(for: newCoin)
      }
```